jenkins/e2e-image/Dockerfile

```diff
@@ -52,4 +52,25 @@ ADD ["e2e-runner.sh", \
     "/workspace/"]
 RUN ["chmod", "+x", "/workspace/get-kube.sh", "/workspace/upload-to-gcs.sh", "/workspace/sh2ju.sh"]
 
+# Add kubernetes-anywhere dependencies: jsonnet, terraform
+ENV TERRAFORM_VERSION 0.7.2
+
+RUN apt-get update && \
```

Comment:
maybe we should do all of this before the `ADD ["e2e-runner.sh" ...]` etc, since we expect the shell scripts to change frequently, and these dependencies to be fairly stable.